src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/SubmissionComponentSpec.scala

```diff
@@ -75,7 +87,10 @@ class SubmissionComponentSpec extends TestDriverComponentWithFlatSpecAndMatchers
 
     runAndWait(submissionQuery.updateStatus(workspaceContext, testData.submission1.submissionId, SubmissionStatuses.Done))
 
-    assertResult(Some(testData.submission1.copy(status = SubmissionStatuses.Done))) {
+    // for performance reasons, input resolutions are not returned when listing workflows inside a submission
+    val expectedWorkflows = testData.submission1.workflows.map{_.copy(inputResolutions=Seq.empty)}
```

Comment:
Yeah agreed it's a bit awkward to have to do all this in the unit tests to support overloading the use of Workflow rather than making a different case class.  If we are fine on maintaining the tests this way or accruing the tech debt later after the fire, then this seems fine to me.